docs/spec_draft.md

```diff
@@ -438,16 +438,16 @@ tensor. More formally,
 
 ### Inputs
 
-| Name        | Type                                              |
-|-------------|---------------------------------------------------|
-| `inputs`    | variadic number of tensors of any supported types |
-| `dimension` | `si64`                                            |
+| Name        | Type                                                      |
+|-------------|-----------------------------------------------------------|
+| `inputs`    | variadic number of tensors of any supported element types |
```

Comment:
Also what do you think about the wording "of any supported type" (i.e. singular vs plural)? As I was reading TCP design docs today (e.g. [this one](https://docs.google.com/document/d/1x8kCUsYzFEIeWO38h32bHlmDrieeu9UJTxpKe9gUm_g/edit#heading=h.ey5ld3h9qkk2)), I noticed that they use the signular form, and I think it reads better - "tensor of supported types" might suggest that there may be multiple different types involved.

Reply:
makes sense to me.